tools/KeyfileIOUtils.cpp

```diff
@@ -20,12 +20,15 @@
 #include <exception>
 #include "KeyfileIOUtils.hpp"
 #include "yaml_utils.hpp"
+#include "util/filesystem.hpp"
 
 void outputReplicaKeyfile(uint16_t numReplicas,
                           uint16_t numRoReplicas,
                           bftEngine::ReplicaConfig& config,
                           const std::string& outputFilename,
                           Cryptosystem* commonSys) {
+  std::cout << "Outputting key file: " << fs::absolute(outputFilename) << std::endl;
```

Comment:
Adding prints to stdout may break tests that assume a specific output.
I would remove this line.

Reply:
I added this because key generation is voodoo.
ConcordKeyGenerator is a separate process, who parses its output?

Reply:
Maybe no one, but we had issues before where other parts of the product parsed the output (maybe config service?).
Hence I would prefer to avoid printing to the stdout unless it is absolutely needed

Reply:
I switched over to use the info log.
I tend to disagree, if the code passes CI.